### PR TITLE
using an icon instead of text for checkmark

### PR DESCRIPTION
**Pull request by DanGodfrey**

### CODE REVIEW

**Comment by 6** [comment]
:shipit: 

we should probably eventually come up with a way to autogenerate `_data-uri-images.scss` from the actual image sources

File: src/form/content-selector/_ws-content-selector.scss [hunk 1]
```diff
@@ -16,14 +16,11 @@
...
       &:after {
-        content:          "✓";
-        background-color: $color-white;
-        padding:          1px 5px;
-        border-radius:    50%;
-        width:            20px;
-        height:           20px;
-        display:          block;
-        font-size:        15px;
+        content:          ".";
```
Comment (6) [inline src/form/content-selector/_ws-content-selector.scss]: what does the period here do?

Comment (DanGodfrey) [inline src/form/content-selector/_ws-content-selector.scss]: certain browsers will not render the pseudo element if the content is empty. This paired with transparent font color tricks it into including it.

**Comment by DanGodfrey** [comment]
That would be awesome, good call.

**Pull request merged**